aten/src/ATen/native/native_functions.yaml

```diff
@@ -128,11 +128,30 @@
 
 - func: add(Tensor self, Tensor other, *, Scalar alpha=1) -> Tensor
   variants: function, method
+  dispatch:
+    CPU: add
+    CUDA: add
+    SparseCPU: add
+    SparseCUDA: add
+    MkldnnCPU: mkldnn_add
 
 - func: add_(Tensor(a!) self, Tensor other, *, Scalar alpha=1) -> Tensor(a!)
   variants: method
+  dispatch:
+    CPU: add_
+    CUDA: add_
+    SparseCPU: add_
+    SparseCUDA: add_
+    MkldnnCPU: mkldnn_add_
 
 - func: add(Tensor self, Tensor other, *, Scalar alpha=1, Tensor(a!) out) -> Tensor(a!)
+  matches_jit_signature: True
```

Comment:
no need to put matches_jit_signature

Reply:
yes, please remove this, I just spent a bunch of time killing all of these :).